sync-todo/v2/client/maui/RealmTodo/AppShell.xaml

```diff
@@ -16,5 +16,10 @@
         ContentTemplate="{DataTemplate local:ItemsPage}"
         Route="items" />
 
+    <ShellContent
```

Comment:
Out of curiosity, could we not also define it in the xaml as a nested route, like this?
```
<ShellContent
        Title="RealmTodo"
        ContentTemplate="{DataTemplate local:ItemsPage}"
        Route="items" >
        <ShellContent
            Title="RealmTodo"
            ContentTemplate="{DataTemplate local:EditItemPage}"
            Route="editItem" />
    </ShellContent>
```

Reply:
In this case it could work, but it's a little different. If you define it in the xaml, then it means that if you show EditItemPage, then there must be ItemsPage just behind.
If you define it in code, you're creating a "global routes", that can be called from anywhere in the app. This is generally useful for "detail pages". For example, in a todo app you maybe want to be able to see/modify the todo from different parts of the application.